test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -340,6 +304,35 @@ public void ParameterHasFieldPrefix([Bind(Prefix = "simpleModelPrefix")] string
 
             // Assert
             Assert.Equal("Hello", controller.ValueBinderMarkedProperty);
+            Assert.Equal(new List<string> { "goodbye" }, controller.CollectionValueBinderMarkedProperty);
+            Assert.Null(controller.UnmarkedProperty);
```

Comment:
For all properties where we are not attempting to bind like `CollectionValueBinderMarkedProperty` and `UnmarkedProperty` I am wondering should we just remove these validation to simply and focus the test?